integration_tests/src/main/python/prune_partition_column_test.py

```diff
@@ -127,3 +128,78 @@ def test_prune_partition_column_when_filter_fallback_project(spark_tmp_path, pru
                                                              filter_col, file_format):
     do_prune_partition_column_when_filter_project(spark_tmp_path, prune_part_enabled, file_format,
                                                   filter_col, gpu_project_enabled=False)
+
+def create_contacts_table_and_read(is_partitioned, format, data_path, expected_schemata, func, conf):
```

Comment:
could we get some comments or a doc string explaining this? It is named in a generic way, but it is not 100% clear what it is intended to do.